### PR TITLE
Cut version `1.0.0`

### PR DESCRIPTION
**Pull request by harrylewis**
# Description

This PR cuts version `1.0.0` of this library.

# What To Test

- [x] CI passes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,7 +294,7 @@ expected to adhere to the [Contributor Covenant](https://www.contributor-covenan
 
 ## License
 
-The gem is available as open source under the terms of the MIT License.
+The gem is available as open source under the terms of the attached Apache 2.0 license.
```
Comment (syedahmedhussaini) [inline README.md:297]: So for the MIT license I think _technically_ you're supposed to keep the original in (even when it changes). Something like this:

`Apache 2.0 License`
`Copyright 2023 Lillio`
`The gem is available as open source under the terms of the attached Apache 2.0 license.`

`MIT License`
`Copyright (c) 2020 HiMama`
`The gem was originally available as open source under the terms of the MIT License.`

Comment (harrylewis) [reply README.md:297]: The license we have here is an Apache 2.0 license, copyrighted to `2020 Payrix`.

https://github.com/HiMamaInc/payrix/blob/5dd9fe7ad82afe371d8e24470e1150ec6e86d21d/LICENSE#L189

Do we want to change this?

Comment (syedahmedhussaini) [reply README.md:297]: Yea can change this to Lillio!

**Review by syedahmedhussaini — APPROVED** [review]
Looks good to me. License agreement stuff can be patched in.

**Pull request merged**